app/src/ui/publish-repository/publish.tsx

```diff
@@ -37,22 +70,11 @@ interface IPublishState {
   /** The currently selected tab. */
   readonly currentTab: PublishTab
 
-  /** The settings for publishing the repository. */
-  readonly publishSettings: IPublishRepositorySettings
+  /** The state of dotCom tab. */
+  readonly dotcomTabState: IDotcomTabState
 
-  /**
-   * An error which, if present, is presented to the
-   * user in close proximity to the actions or input fields
-   * related to the current step.
-   */
-  readonly error: Error | null
-
-  /**
-   * The error pertaining to the tab the user is not
-   * actively using. It is swapped when the user changes
-   * the current tab.
-   */
-  readonly inactiveTabError: Error | null
+  /** The state of enterprise tab. */
+  readonly gheTabState: IEnterpriseTabState
```

Comment:
I'd love for this to be named `enterpriseTabState` (as well as the usages) so these comments are unnecessary